src/System/System.php

```diff
@@ -79,8 +79,6 @@ public static function getOS(): string
 
     /**
      * Returns the architecture of the system's processor.
-     *
-     * @return string
```

Comment:
let us keep all the return comment docs from function declaration as functions are returning values

Reply:
Fixed